src/openrct2/park/Legacy.h

```diff
@@ -26,6 +27,390 @@ namespace OpenRCT2
     {
         struct FootpathMapping;
     }
+
+    struct ConvertedTrackTypeResult
+    {
+        TrackElemType trackType;
+        bool isCovered;
+    };
+
+    enum class OldTrackElemType : uint16_t
```

Comment:
New TrackElemTypes do not have to be added to this list, since it's only called on parks older than this park version.

Reply:
I’m confused. Does it say this anywhere?

Reply:
It doesn't say it, that's why I'm saying it.